src/server/license/check-answers/index.njk

```diff
@@ -0,0 +1,34 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% extends 'layouts/questions.njk' %}
+
+{% block questions %}
+  <h1 class="govuk-heading-l">{{pageTitle}}</h1>
+
+  {{ govukSummaryList({
+    classes: "govuk-!-margin-bottom-9",
+    rows: [
+      {
+        key: {
+          text: "What email address would you like the licence sent to?"
+        },
+        value: {
+          text: license.emailAddress
```

Comment:
this should be the html() method of the `emailAddress` - the one that's designed for rendering a summary, not necessarily for playing back into an input field.